install/kubernetes/quick-install.yaml

```diff
@@ -482,6 +482,7 @@ spec:
         - mountPath: /var/run/cilium
           name: cilium-run
       restartPolicy: Always
+      priorityClassName: system-node-critical
```

Comment:
This needs to be removed in order to be compatible with all versions of k8s. I'll add a PR to run a `sed` to remove it so we don't depend on the k8s version context `helm` is in when a developer runs it.